Gemfile.lock

```diff
@@ -1,214 +1,240 @@
 PATH
   remote: .
   specs:
-    subserver (0.5.0)
-      google-cloud-pubsub (>= 1.7, < 3.0)
+    subserver (0.5.1)
+      google-cloud-pubsub (>= 2.0, < 3.0)
```

Comment:
There were errors trying to find a matching `google-protobuf` gem in the spec. Bumping the min to 2.0 resolved it. Looks like `churchonline-rb` has it locked @ 2.5 anyways.